modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -66,6 +66,7 @@ export type Tileset3DProps = {
 
   // Traversal
   maximumScreenSpaceError?: number;
+  memoryAdjustedScreenSpaceError?: boolean;
```

Comment:
Nit: doc comment please

Reply:
The comment is on line 110 with other documentation of `type Props`. Do you think it should be copied here too?

Reply:
Hmm. at the moment I am not sure why we have two copies. There was a scramble to add types to these modules for loaders.gl 4.0 and there may be some cleanup work left.

I will say that many devs use vscode and it is nice to be able to hover in vscode and see these comments, so I try to make sure all types and fields are fully commented, but if it doesn't make sense.

Reply:
OK Yeah I see now there is a redundancy here. I could try to consolidate these two types? It seems that the `Props` type is not being exported but only used to define `DEFAULT_PROPS`

Reply:
That would be nice, best done in a separate PR. 

Not sure if there are any complications, it may be a hack to accommodate an I3S subclass or something like that.